editor-components/call_to_action/call_to_action6/call_to_action6.tsx

```diff
@@ -55,36 +77,73 @@ class CallToAction6Page extends BaseCallToAction {
   }
 
   render() {
+    const title = this.getPropValue("title", { as_string: true });
+    const placeholder = this.getPropValue("placeholder", { as_string: true });
+    const comment = this.getPropValue("comment", { as_string: true });
+
+    const button = this.castToObject<{
+      button_text: JSX.Element;
+      link: string;
+    }>("button");
+
+    const icon = this.getPropValue("icon");
+
+    const isButtonEmpty = icon || this.castToString(button.button_text);
```

Comment:
this shouldn't be like     const isButtonEmpty = icon || !this.castToString(button.button_text);